build.sbt

```diff
@@ -1,29 +1,26 @@
 import ReleaseTransformations._
+import sbtversionpolicy.withsbtrelease.ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease
 
 name := "tags-thrift-schema"
 organization := "com.gu"
 
-scmInfo := Some(ScmInfo(url("https://github.com/guardian/tags-thrift-schema"), "scm:git@github.com:guardian/tags-thrift-schema"))
-homepage := scmInfo.value.map(_.browseUrl)
-developers := List(Developer(id = "guardian", name = "Guardian", email = null, url = url("https://github.com/guardian")))
-
-
 libraryDependencies ++= Seq(
   "org.apache.thrift" % "libthrift" % "0.13.0",
   "com.twitter" %% "scrooge-core" % "19.11.0"
 )
 
 // Might cross compile more scala versions here depending on who needs this!
-scalaVersion := "2.13.1"
-crossScalaVersions := Seq("2.11.12", "2.12.10", scalaVersion.value)
+scalaVersion := "2.13.11"
+crossScalaVersions := Seq("2.12.18", scalaVersion.value)
+scalacOptions := Seq(
+  "-release:11"
+)
 releaseCrossBuild := true
 
 resolvers += Resolver.jcenterRepo
 
 licenses += ("Apache-2.0", url("https://github.com/guardian/tags-thrift-schema/blob/main/LICENSE"))
-
-releasePublishArtifactsAction := PgpKeys.publishSigned.value
-publishTo := sonatypePublishTo.value
+releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value
 
 releaseProcess := Seq[ReleaseStep](
```

Comment:
The steps in `releaseProcess` need to be updated - see [configuration.md#recommended-sbt-settings](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/docs/configuration.md#recommended-sbt-settings) and the [example file ](https://github.com/guardian/etag-caching/blob/8528412e1b1fc24079345a01f416d015bdcd5196/build.sbt#L59-L69)- we just want `checkSnapshotDependencies, inquireVersions, runClean, runTest, setReleaseVersion, commitReleaseVersion, tagRelease, setNextVersion, commitNextVersion`

Reply:
Thanks, I had that change locally but somehow missed committing it